drivers/sensor/sbs_gauge/sbs_gauge.c

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sbs_gauge
+
+#include <drivers/i2c.h>
+#include <drivers/sensor.h>
+#include <sys/byteorder.h>
+
+#include "sbs_gauge.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(sbs_gauge, CONFIG_SENSOR_LOG_LEVEL);
+
+static int sbs_cmd_reg_read(const struct device *dev, uint8_t reg_addr,
+				    int16_t *val)
+{
+	const struct sbs_gauge_config *const cfg;
```

Comment:
Apply https://github.com/jfischer-no/zephyr/commit/09925db46958993cecef4a6b21ce613b287016df.patch

Reply:
The indention and the spaces/tabs should be now according your patch, please review. Thank you.